docs/hardware/devices/index.mdx

```diff
@@ -245,6 +245,19 @@ nRF52840-based development boards with GPS, and multiple ports to attach sensors
 | [Seeed Wio-WM1110 Dev Kit](./seeed-studio/wm1110?wm1110=wio-sdk-wm1110)   | nRF52840 | LR1110 | YES  | 5.3 | YES |
 | [Seeed Wio Tracker 1110](./seeed-studio/wm1110?wm1110=wio-tracker-wm1110) | nRF52840 | LR1110 | YES  | 5.3 | YES |
 
+
+## [PSMG smart glasses by Seeed] (coming soon)
+
+nRF52840-based development boards, and multiple ports to attach sensors via seeed grove connectors.
+
+
+| Name                                                                      | MCU      | Radio  | WiFi | BT  | GPS      |
+| :------------------------------------------------------------------------ | :------- | :----- | :--: | :-: | :-:      |
+| [Seeed esp32-sx1262-kit](./seeed-studio/wm1110?wm1110=wio-sdk-wm1110)     | nRF52840 | LR1110 | YES  | 5.3 | OPTIONAL |
+
+🕶️ PMSG: Dev-ready wearable with LoRa, WiFi, BT, (optional) OLED, motion sensors, vibration motor & LiPo power+ grove ports for more
```

Comment:
I'm having a hard time figuring out where this description should go...the one above it seems to be a copy/paste from the wm1110 from above. Should this go in that spot and replace it? Either way none of the others seem to have descriptions below them like this so we should move it above and replace to add to what is already there. I think.